src/renderer/components/reader/ReaderMenu.tsx

```diff
@@ -195,9 +219,55 @@ export class ReaderMenu extends React.Component<Props, State> {
         return undefined;
     }
 
+    private buildGoToPageSection() {
+        const { __ } = this.props;
+        const error = this.state.pageError;
+        return <div className={styles.goToPage}>
+            <p className={styles.title}>{__("reader.navigation.goToTitle")}</p>
+            {error &&
+                <p
+                    className={styles.goToErrorMessage}
+                    aria-live="assertive"
+                    aria-relevant="all"
+                    role="alert"
+                >
+                    { __("reader.navigation.goToError") }
+                </p>
+            }
+            <form onSubmit={this.handleSubmitPage}>
+                <input
+                    ref={(ref) => this.goToRef = ref}
+                    type="number"
```

Comment:
This should be a text input field so that users can enter any arbitrary string, such as Roman numerals (e.g. `ii`).
Also, `type="number"` allows negative values which is weird :)

Reply:
Furthermore, the built-in HTML validation feature does not seem to take into account the document language (HTML `lang` attribute), resulting in locale mismatch (see screenshot below with exponent notation).

Reply:
![ThoriumGotoPageInputValidationNumber](https://user-images.githubusercontent.com/625201/64261939-40d5be00-cf25-11e9-94c2-0b0da71b4951.png)